internal/cli/apply_test.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestApply(t *testing.T) {
+	f := newServerFixture(t)
+	defer f.TearDown()
+
+	f.WriteFile("sleep.yaml", `
+apiVersion: tilt.dev/v1alpha1
+kind: Cmd
+metadata:
+  name: my-sleep
+spec:
+  args: ["sleep", "1"]
+`)
+	out := bytes.NewBuffer(nil)
+
+	cmd := newApplyCmd()
+	cmd.streams.Out = out
+	c := cmd.register()
+	err := c.Flags().Parse([]string{"-f", f.JoinPath("sleep.yaml")})
+	require.NoError(t, err)
+
+	err = cmd.run(f.ctx, nil)
+	require.NoError(t, err)
+	assert.Contains(t, out.String(), `cmd.tilt.dev/my-sleep created`)
```

Comment:
Similar to delete, might be worthwhile to just check that `f.client.Get()` can find `my-sleep`

Reply:
done